docs/rfcs/0002/readme.md

```diff
@@ -37,21 +34,58 @@ The proposal below might take a couple of read-throughs; I've also added a concr
+    3. The containerized application(s) executes. The expected mounts are available per the requirements and celebration occurs. The pods continue to run until:
+       1. the pod completes successfully (any failed pods will be restarted)
+       2. timeout (i.e. `activeDeadlineSeconds`) is hit (optional)
+       3. the max number of pod retries (i.e. `backoffLimit`) is hit (indicating failure on all retry attempts)
+          1. Note: retry limit is non-optional per Kubernetes configuration
+          2. If retries are not desired, this number could be set to disable any retry attempts.
+    4. `PostRun`: RABBIT software:
+       1. Mark the stage as `Ready` if the pods have all completed successfully. This includes any retries after preceding failures.
+       2. If all pods are completed but did complete successfully, the stage is not be marked as ready.
+       3. Leave all pods around for log inspection
+
+### Communication Details
+
+Other than mounts, the following subsections outline the proposed communication between the Rabbit nodes themselves and the Compute nodes.
+
+#### Rabbit-to-Rabbit Communication
+
+A headless Kubernetes service connects the pods. This service is unique to each container workflow. Each rabbit node would be
+reached via `<host-name>.<service-name>`. The service-name would be provided to the application via a well-known environmental variable.
+This has been prototyped and has proven to be successful. The list of Rabbits available for communication is provided as well.
+
+#### Compute-to-Rabbit Communication
+
+For Compute to Rabbit communication, the proposal is to use an open port between the nodes, so the applications could communicate using IP protocol.
+The port number would be assigned by the Rabbit software and included in the workflow resource's environmental variables after the Setup state (similar to workflow name & namespace).
+Flux should provide the port number to the compute application via an environmental variable or command line argument. The containerized application
+would always see the same port number using the `hostPort`/`containerPort` mapping functionality included in Kubernetes. To clarify, the Rabbit software is picking
+and managing the ports picked for `hostPort`.
+
+This requires a range of ports to be open in the firewall configuration and specified in the rabbit system configuration.
+The fewer the number of ports available increases the chances of a port reservation conflict that would fail a workflow.
+
+For safe port reusability, this port range must be large enough to account for the anticipated number of containerized jobs running concurrently. This allows time for the Linux kernel
+to prepare a port for reuse. The kernel must take this port range into consideration when defining the ephemeral port range.
+
+#### Rabbit-to-Compute Communication
+
+Same approach as above.
```

Comment:
If you consult the email chain, Rabbit-to-Compute communication, where the compute node would provide some TCP/IP server and a Rabbit client could contact it, is _not_ supported.

Reply:
That email chain is a mess and I might be missing emails since I didn't get all the them in the first place.

So just say this is not supported?

Reply:
My emails aren't that poorly written, are they?
> **Rabbit to Compute communication**
> With the proposal above, the containerized application running on the Rabbit would act as a server and the compute nodes as clients. Is this sufficient, or do you require the reverse for Rabbit to Compute communication?

I didn't get a direct answer, but took that as "We do **not** need Rabbit to Compute"